doc/userguide/rules/payload-keywords.rst

```diff
@@ -278,6 +278,18 @@ A negated isdataat on an absent buffer will match any value.
 
```

Comment:
please explain both idea and high level implementation in the commit message

Reply:
ok trying to improve the commit message